README.md

```diff
@@ -132,7 +141,7 @@ from babbage.manager import JSONCubeManager
 from babbage.api import configure_api
 
 app = Flask('demo')
-engine = 
+engine =
```

Comment:
I just noticed, my editor took all trailing whitespace off lines automatically. That's what all these little changes are.